src/share.js

```diff
@@ -68,7 +68,7 @@ class Share extends BasePlugin {
     this.player.ready().then(() => {
       const ShareWrapper = () => <ShareComponent config={this.config} />;
       this.iconId = this.player.getService('upperBarManager').add({
-        label: pluginName,
+        label: 'Share',
```

Comment:
it is very problematic if different systems in our ecosystem are using different annotation.
if all plugins have lower case naming we should handle this on the upperBarManager config